timemachine/exchange/exchange_mover.py

```diff
@@ -61,12 +63,15 @@ def build_system(host_pdbfile: str, water_ff: str, padding: float):
     return solvated_host_system, solvated_host_coords, box, solvated_topology, nwa
 
 
-# currently un-used.
 def randomly_rotate_and_translate(coords, offset):
+    # coords: shape 3(OHH) x 3(xyz), water coordinates
     centroid = np.mean(coords, axis=0, keepdims=True)
     centered_coords = coords - centroid
     rot_mat = special_ortho_group.rvs(3)
-    rotated_coords = np.matmul(centered_coords, rot_mat)
+
+    # equivalent to standard form where R is first:
+    # (R @ A.T).T = A @ R.T
+    rotated_coords = centered_coords @ rot_mat.T
```

Comment:
could shrink slightly using [scipy spatial Rotation](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html), something like

```python
rotation = Rotation.random()
rotation.apply(centered_coords)
```

depending on how much reminder of low-level details we want in this function

Reply:
I'm going to keep it as-is for now, since I'll be porting this function directly to C++ (where I care about low-level details).